pool_configs/vot.json

```diff
@@ -0,0 +1,87 @@
+{
+    "enabled": false,
+    "coin": "vot.json",
+
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "walletInterval": 2.5,
+
+    "rewardRecipients": {
+        "": 0.2
+    },
+
+    "paymentProcessing": {
+	"minConf": 10,
```

Comment:
Can  you fix the alignment on this line?

Reply:
done.